### DISCUSSION TITLE
B站播放又出新版了、累

### DISCUSSION THREAD

**Discussion by Malus-Mill** [the1812/Bilibili-Evolved#3340] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/53999738/169506854-3428dec7-253f-4605-9aff-bb8fac2a0dd3.png)
![image](https://user-images.githubusercontent.com/53999738/169506890-7cfab50f-ac1d-48b9-a10a-4bdaf42b022d.png)
![image](https://user-images.githubusercontent.com/53999738/169506921-727fde4b-0992-4099-9f04-f419beb4e71c.png)

**Comment by the1812** [comment]
#2615

**Comment by kethoven** [comment]
看不出变化啊。我还以为又要大改了。
